adapter/internal/oasparser/envoyconf/envoyconf_internal_test.go

```diff
@@ -46,19 +46,16 @@ func TestGenerateRoutePaths(t *testing.T) {
 	basePath := "/basePath"
 	resourcePath := "/resource"
 
-	completeRoutePath := generateRoutePaths(xWso2BasePath, basePath, resourcePath)
+	filteredBasePath := getFilteredBasePath(xWso2BasePath, basePath)
+	completeRoutePath := generateRoutePath(filteredBasePath, resourcePath)
```

Comment:
I guess we can add a unit test to route path when default version is given. maybe in routes_with_clusters_test.go